internal/conditions/conditions.go

```diff
@@ -47,6 +47,14 @@ const (
 	ReasonMetricAgentNotRequired = "AgentNotRequired"
 )
 
+// Error messages
+const (
+	podIsNotScheduled    = "Pod not scheduled: %s"
+	podIsPending         = "Pod is in pending state as container: %s is not running due to: %s"
+	podIsFailed          = "Pod is in failed state due to: %s"
+	podRolloutInProgress = "Pods are being started/updated"
+)
```

Comment:
lets make this a bit more uniform:
either Pod is not scheduled, or we remove the verb from the others...